tart2ms/ms_helper.py

```diff
@@ -65,6 +84,70 @@ def get_observation_time(ms_file):
             "height": geo.height.to_value('m')
             }
 
+def get_catalog_sources_azel(timestamps, location):
+    catsources = catalog_factory.from_3CRR(fluxlim15=0.5)
+    catsources += catalog_factory.from_SUMMS(fluxlim15=0.5)
+    catsources += catalog_factory.from_MKGains(fluxlim15=0.5)
+    sources = []
+    ras = list(map(lambda s: s.rarad, catsources))
+    decs = list(map(lambda s: s.decrad, catsources))
+    alts, azs = radec2azel(ras, decs, location=location, obstime=timestamps)
+    for ti, tt in enumerate(timestamps):
+        ttsources = []
+        for el, az, ss in zip(alts[ti], azs[ti], catsources):
+            name = ss.name
+            ttsources.append({
+                "name": f"CELESTIAL_{name}",
+                "flux": ss.flux,
+                "az": np.rad2deg(az),
+                "el": np.rad2deg(el),
+            })
+        sources.append(ttsources)
+        
+    return sources
+
+def get_solar_system_bodies(timestamps, location):
+    # BH: only adding a few of the brightest right now
+    # flux is estimated by RJ -- may be wildly off for the sun
+    # if it flares
+    class blackbody:
+        radius = {
+            "Moon": 1738, #km
+            "Sun": 696340 #km
+        }
+        blackbody_temp = {
+            "Moon": 270, # avg K
+            "Sun": 5778 # avg K
+        }
+        def __init__(self, name, distance):
+            self.__name = name
+            self.__distance = distance
+        
+        def rj(self):
+            """ RJ as a function of angular size (amin) and temperature (K) """
```

Comment:
This is just a low frequency approximation for temperature to Jy for a black body radiator

Reply:
We need to move this to a separate set of files. ms_helper is getting too big.